R/tt_dotabulation.R

```diff
@@ -1250,8 +1250,12 @@ setMethod("set_def_child_ord", "VarLevWBaselineSplit",
             vals <- levels(relevel(droplevels(vals), bline))
         }
     }
-
-    stopifnot(bline %in% vals)
+    if (!bline %in% vals) {
+        stop(paste(
```

Comment:
Please include both the missing ref group (`bline`) and the variable (`spl_payload(lyt)`, as you already have) in the error message.

Reply:
I've added `bline` to the error message.